app/src/main/resources/swagger/api-docs-v1.json

```diff
@@ -4525,6 +4525,71 @@
         } ]
       }
     },
+    "/users/{id}/update" : {
```

Comment:
Here are a few ideas:

1. Bad description: If sending the notification is a side effect, I would change the description to "update user's data and send a notification to...< whom??? >." The name of this operation, "updateUserUsingPOST," suggests that this API is performing a User resource update, but the description instead states that this API will send a notification to the User.
2. which kind of notification are we sending? email to user? event to datalake? I'd make it clear in the description
3. We have a parameter called "institutionId," but the description does not make it apparent what it is for, I'd add the purpose of this field into the API description
4. Why not just PUT to "/users/id" if this is a change to a user resource? if this API just send a notification, a POST to /users/{id}/notifications would be better (try to use resources name instead of actions (ie. "update") 
5. optional: ideally notifications are resources you can query in a later time to get their status

Consider the OpenAPI specifications to be the primary documentation for the service; later developers who contribute to the project should be able to comprehend what is going on.